test/Microsoft.AspNet.Mvc.FunctionalTests/ActionResultTests.cs

```diff
@@ -8,35 +8,33 @@
 using System.Net.Http.Headers;
 using System.Text;
 using System.Threading.Tasks;
-using ActionResultsWebSite;
-using Microsoft.AspNet.Builder;
-using Microsoft.Framework.DependencyInjection;
 using Xunit;
 
 namespace Microsoft.AspNet.Mvc.FunctionalTests
 {
-    public class ActionResultTests
+    public class ActionResultTests : IClassFixture<MvcFixture<ActionResultsWebSite.Startup>>
```

Comment:
We could create one fixture per test type - this is a lazier way of going about it.